src/components/OcIcon.vue

```diff
@@ -238,15 +186,6 @@ export default {
       </oc-tbody>
     </oc-table-simple>
 
-    <h3 class="uk-heading-divider">
-      Icons loaded via URL
-    </h3>
-    <div class="oc-m">
-      <oc-icon size="medium" url="https://interactive-examples.mdn.mozilla.net/media/examples/firefox-logo.svg" accessible-label="Firefox logo"/>
-      <oc-icon size="large" url="https://interactive-examples.mdn.mozilla.net/media/examples/firefox-logo.svg"/>
-      <oc-icon size="large" name="account_circle" url="https://interactive-examples.mdn.mozilla.net/media/examples/firefox-logo.sv" accessible-label="Account"/>
```

Comment:
Could you keep this line but get rid of the url attribute? would be nice to keep an example showing the `accessible-label` property. + copy the same line and without accessible-label.

Reply:
Sure!